plugins/inputs/ctrlx_datalayer_sse/sample.conf

```diff
@@ -0,0 +1,124 @@
+   ## It is possible to define multiple subscriptions per host. Each subscription can have its own
+   ## sampling properties and a list of nodes to subscribe to.
+   ## All subscriptions share the same credentials.
+   [[inputs.ctrlx_datalayer_sse.subscription]]
+      ## The name of the measurement. (default: "ctrlx")
+      measurement = "metrics"
+
+      ## Define an array of nodes with field name and node address
+      ## name - field name to use in the output (optional, default: "value_{type}")
+      ## address - node address in ctrlX Data Layer
+      ## Example:
+      ## {name="counter", address="scheduler/admin/info/counter"}
+      nodes=[
+         {name="cpu_usage_percent", address="framework/metrics/system/cpu-utilisation-percent"},
+         {name="memory_available", address="framework/metrics/system/memavailable-mb"},
+         {name="memory_used", address="framework/metrics/system/memused-mb"},
+         {name="memory_used_percent", address="framework/metrics/system/memused-percent"},
+      ]
+
+      ## The switch "output_json_string" determines the format of the output metric. (default: false)
+      ## false: the default output format is used with the following conversion rule:
+      ##  - simple types (bool8,int8,uint8,int16,uint16,int32,uint32,int64,uint64,float,double,string,timestamp): 
+      ##    pass the value as it ist to a metric with pattern '{name}={value}'
+      ##  - array types (arbool8,arint8,aruint8,arint16,aruint16,arint32,aruint32,arint64,aruint64,arfloat,ardouble,arstring,artimestamp): 
+      ##    pass every value in the array to a metric with pattern '{name_index}={value[index]}'
+      ##    example: myarray=[1,2,3] -> myarray_1=1, myarray_2=2, myarray_3=3
+      ##  - object types (json types): 
+      ##    use a json flattener and pass every flattened value to a metric with pattern '{name}_{jsonflat.key}={jsonflat.value}'
+      ##    example: myobj = {"a":1,"b":2,"c":{"d": 3}} -> myobj_a=1, myobj_b=2, myobj_c_d=3
+      ## true: the output value is formatted to json string
+      ##    pass the value as json string to a metric with pattern '{name}="{value}"'
+      ##    examples: 
+      ##      input = true -> output = "true"
+      ##      input = [1,2,3] -> output = "[1,2,3]"
+      ##      input = {"x":4720,"y":9440,"z":{"d": 14160}} -> output = "{\"x\":4720,\"y\":9440,\"z\":14160}"
+      ##    output json string can be used for further processing e.g. starlark.
+      ##    see: https://docs.influxdata.com/telegraf/v1.23/plugins/#processor-plugins
+      ##    example:
+      ##      [[inputs.ctrlx_datalayer_sse.subscription]]
+      ##      measurement = "osci"
+      ##      nodes = [
+      ##        {address="oscilloscope/instances/Osci_PLC/rec-values/allsignals"},
+      ##      ]
+      ##      output_json_string = true
+      ##      [[processors.starlark]]
+      ##      namepass = [
+      ##         'osci',
+      ##      ]
+      ##      script = "oscilloscope.star"
```

Comment:
Please move this lengthy description to the `README.md` and only provide a short summary here.